CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.5.0
+ - Host settings now are more robust to bad input and can take (almost) full URLs
```

Comment:
almost?

Also, I would mention the URL part first since that's the major change. Possibly make this two entries; one about URLs acceptance, and one about the input validation